app.js

```diff
@@ -186,3 +189,22 @@ function syncToTarget (event, target) {
     server.replicateNetwork(socket, 'pull')
   }
 }
+
+function handleUncaughtExceptions () {
+  process.on('uncaughtException', function (error) {
+    // TODO(noffle): log to logging mechanism once
+    // https://github.com/digidem/mapeo-desktop/issues/77 is in.
+    console.log('uncaughtException', error)
+
+    // Show a vaguely informative dialog.
+    var opts = {
+      type: 'error',
+      buttons: [ 'OK' ],  // TODO: internationalize
```

Comment:
We get away with it for now, "OK" is the same in Spanish, and "Fatal Error" is close enough to "Error Fatal" for people to understand...